modules/60crypt-ssh/module-setup.sh

```diff
@@ -106,7 +106,10 @@ install() {
   inst_hook pre-udev 99 "$moddir/dropbear-start.sh"
   inst_hook pre-pivot 05 "$moddir/dropbear-stop.sh"
 
+  #install the authorized_keys file and fix the ownership/permission
   inst "${dropbear_acl}" /root/.ssh/authorized_keys
+  chmod 400 ${initdir}/root/.ssh/authorized_keys
+  chown root:root ${initdir}/root/.ssh/authorized_keys
```

Comment:
It's better to use `0:0` instead of `root:root` here, in case some weird environment doesn't map `root` to uid 0.